person.rb

```diff
@@ -1,16 +1,17 @@
 require './nameable'
 
 class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [ ] Great job, here we need both setters and getters for rentals. Kindly implement that.